airflow/config_templates/config.yml

```diff
@@ -1028,6 +1028,27 @@ secrets:
       type: string
       example: ~
       default: ""
+    use_cache:
+      description: |
+        Enables local caching of Variables, when parsing DAGs only.
+        Using this option can make dag parsing faster if Variables are used in top level code, at the expense
+        of longer propagation time for changes.
+        Please note that this cache concerns only the DAG parsing step. There is no caching in place when DAG
+        tasks are run.
+      version_added: 2.7.0
```

Comment:
not too sure about this one... if it gets merged in a reasonable time frame, my guess is that it'll land there ?

Reply:
If it gets merged now - it will land in 2.6.0

Reply:
(And I think we should aim for it)

Reply:
well that'd be cool. I'll change it to 2.6 then, we'll always have time to change it back if this drifts in time.